### PR TITLE
[UR][CUDA][HIP][L0] Cleanup licence header

### PR DESCRIPTION
**Pull request by veselypeta**
Fix the license headers at the top of each source file in the unified runtime directory.

### CODE REVIEW

File: sycl/plugins/unified_runtime/ur/adapters/hip/memory.cpp [hunk 1]
```diff
@@ -1,3 +1,11 @@
+//===--------- memory.cpp - HIP Adapter ---------------------------===//
```
Comment (npmiller) [inline sycl/plugins/unified_runtime/ur/adapters/hip/memory.cpp:1]:
```suggestion
//===--------- memory.cpp - HIP Adapter ------------------------------===//
```

File: sycl/plugins/unified_runtime/ur/adapters/hip/memory.hpp [hunk 2]
```diff
@@ -1,4 +1,4 @@
-//===--------- context.cpp - HIP Adapter ----------------------------===//
+//===--------- memory.hpp - HIP Adapter ----------------------------===//
```
Comment (npmiller) [inline sycl/plugins/unified_runtime/ur/adapters/hip/memory.hpp:1]:
```suggestion
//===--------- memory.hpp - HIP Adapter ------------------------------===//
```

**Review by npmiller — APPROVED** [review]
Small formatting fixes but looks good otherwise

**Review by jchlanda — APPROVED** [review]

**Review by PietroGhg — APPROVED** [review]

File: sycl/plugins/unified_runtime/ur/adapters/hip/memory.cpp [hunk 3]
```diff
@@ -1,3 +1,11 @@
+//===--------- memory.cpp - HIP Adapter ------------------------------===//
```
Comment (bader) [inline sycl/plugins/unified_runtime/ur/adapters/hip/memory.cpp:1]: Please, make the 1st and 7th line exactly 80 characters long in all files.
```suggestion
//===--------- memory.cpp - HIP Adapter -----------------------------------===//
```

**Review by jandres742 — APPROVED** [review]

**Comment by veselypeta** [comment]
@intel/bindless-images-reviewers

**Review by ProGTX — APPROVED** [review]

**Comment by veselypeta** [comment]
@intel/llvm-gatekeepers

**Pull request merged**